app/controllers/facilities_controller.rb

```diff
@@ -113,6 +114,14 @@ def transactions_with_search
     paginate_order_details
   end
 
+  # GET /facilities/:facility_id/disputed_orders
+  def disputed_orders_with_search
+    set_default_start_date
```

Comment:
I think we can remove this call, which should address your concern about the confusion of maybe not seeing an order if it's outside the range. That was mainly there on the main one so CSV exports don't include _everything_ by default.